00 Intro/01 JSX/src/components/App.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { User } from '../entity/User';
+import { PanelComponent } from '../components/panel-component/PanelComponent';
+import { UsersTableComponent } from '../components/user-table-component/UsersTableComponent';
+
+interface State {
+  users: User[];
+}
+
+class App extends React.Component<{}, State> {
```

Comment:
I decided to expose export sentences at the end of the file. Is the any convention I should follow?